src/theme/bootstrap.config.scss

```diff
@@ -0,0 +1,60 @@
+/**
+ * Load each file that ypu need, as model clasic of CSS
+ *
+ */
+$icon-font-path: '~bootstrap-sass/assets/fonts/bootstrap/';
+
+
+// Core variables and mixins
```

Comment:
why not just import the boostrap.scss file that includes all the files under here